chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -535,7 +534,7 @@ def spend_bundle_from_conditions(
 
 async def add_spendbundle(
     mempool_manager: MempoolManager, sb: SpendBundle, sb_name: bytes32
-) -> tuple[Optional[uint64], MempoolInclusionStatus, Optional[Err]]:
+) -> tuple[uint64, MempoolInclusionStatus, Optional[Err]]:
```

Comment:
This is unrelated to the patch, it resulted from an unrelated change below to `add_spendbundle`'s failure path. I've addressed this as well in the addendum.